app/build.gradle.kts

```
@@ -56,4 +56,6 @@ dependencies {
     androidTestImplementation(libs.androidx.ui.test.junit4)
     debugImplementation(libs.androidx.ui.tooling)
     debugImplementation(libs.androidx.ui.test.manifest)
+
+    implementation("androidx.navigation:navigation-compose:2.7.5")
```

toml파일에 
androidx-compose-navigation = { group = "androidx.navigation", name = "navigation-compose", version.ref = "androidxComposeNavigation" }  라고 정의되어 있잖아요.
요거를 가지고
```suggestion
    implementation(libs.androidx.navigation.compose)
```
이렇게 쓸 수 있습니다. toml파일에서 버전 관리를 해주는게 유지보수하기 더 편할거에요.